src-tauri/src/hardware/hardware_status_monitor.rs

```diff
@@ -144,12 +142,11 @@ impl HardwareStatusMonitor {
         }
     }
 
-    async fn load_gpu_devices_from_status_file(&self) -> Result<GpuStatusFileContent, Error> {
-        let config_dir = config_dir().context("Failed to get config directory")?;
-        let file: PathBuf = config_dir
-            .join(APPLICATION_FOLDER_ID)
-            .join("gpuminer")
-            .join("gpu_status.json");
+    async fn load_gpu_devices_from_status_file(
+        &self,
+        config_dir: PathBuf,
+    ) -> Result<GpuStatusFileContent, Error> {
+        let file: PathBuf = config_dir.join("gpuminer").join("gpu_status.json");
```

Comment:
Wouldn't it be better to stick to `dirs::cache_dir()` instead to passing it as parameter as `config_dir` will be always the same

Reply:
i didn't realise it was the same thing 🙀 thanks so much!!

Reply:
Hold on aren't `cache_dir` and `config_dir` different according to [docs](https://docs.rs/tauri/2.1.1/tauri/path/struct.PathResolver.html#method.cache_dir) ?

Reply:
oh yeah :D i did use `dirs::config_dir()` when i made the change. what i didn't realise was that i could use `dirs` to get those paths instead of passing the either the path from `app.path()...` or the whole app handle around :D

Reply:
But I assumed that all three are present in `dirs::`